specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2021-03-01-preview/cosmos-db.json

```diff
@@ -6825,8 +6825,7 @@
         }
       },
       "required": [
-        "resource",
-        "options"
+        "resource"
```

Comment:
can you please check if this is consistent at other places in this file (for SQL, Cassandra, mongo, table, graph dbs and containers)

Reply:
Just checked. It is consistent.